packages/amplify-category-function/package.json

```diff
@@ -9,7 +9,12 @@
     "inquirer": "^7.0.3",
     "mime-types": "^2.1.26",
     "promise-sequential": "^1.1.1",
-    "uuid": "^3.4.0"
+    "uuid": "^3.4.0",
```

Comment:
please make sure referenced version numbers are the latest we've, also linting is consolidated to project level, please remove the scripts below.